src/pages/Detail/component/Creator.jsx

```diff
@@ -0,0 +1,81 @@
+import React, { useState } from 'react';
+import styled from 'styled-components';
+
+function Creator({ creatorName, profile_image }) {
+  const [moreTxt, setMoreTxt] = useState('더보기');
+
+  const moreClick = () => {
+    moreTxt === '더보기' ? setMoreTxt('접기') : setMoreTxt('더보기');
+  };
+
+  return (
+    <>
+      <CreatorIntro>
+        <FlexWrap>
+          <CreatorH3>
+            크리에이터 <br />
+            <strong>{creatorName}</strong> 입니다.
+          </CreatorH3>
+          <CreatorImg src={profile_image} />
+        </FlexWrap>
+
+        <BigCreator moreTxt={moreTxt}>
+          <img src={profile_image} alt="creator_image" />
+        </BigCreator>
+      </CreatorIntro>
+
+      <MoreBtn onClick={moreClick} moreTxt={moreTxt}>
+        {moreTxt}
```

Comment:
```suggestion
        {isOpenContent ? '접기' : '더보기'}
```
state의 변경에 따라 해당 부분에는 이런 삼항연산자가 들어가면 되겠네요!